apps/include/apps.h

```diff
@@ -353,4 +353,6 @@ void app_providers_cleanup(void);
 EVP_PKEY *app_keygen(EVP_PKEY_CTX *ctx, const char *alg, int bits, int verbose);
 EVP_PKEY *app_paramgen(EVP_PKEY_CTX *ctx, const char *alg);
 
+int write_to_file(BIO *in, const char *filename, int format, int private);
+
```

Comment:
Please rename and move in the vicinity of `bio_open_owner()`

Reply:
Updated.

Reply:
Nice that you quickly performed those fixes.

As mentioned, better also move the new function decl within `apps.h` in the vicinity of `bio_open_*`

Reply:
Updated.